Components/Navbar.js

```diff
@@ -1,9 +1,17 @@
+import Link from "next/link";
+
 const Navbar = () => {
   return (
     <nav>
-      <a>Home</a>
-      <a href="/cities">Cities</a>
-      <a href="/about">About Us</a>
+      <Link href="/">
+        <a>Home</a>
```

Comment:
Good to see you wrapping the `<a>` tag, the `<Link>` will work without it but your html will be semantically more correct and more accessible  by using the correct elements as you have here, same for the `<nav>`.

You could actually wrap the {children} in your Layout component in `<main>` tags if you wanted to wanted to keep everything in the correct landmark elements.